charts/activemq-artemis/templates/statefulsets.yaml

```diff
@@ -35,7 +35,7 @@ spec:
       serviceAccountName: {{ template "fullname" . }}
       containers:
       - name: {{ template "fullname" . }}
-        image: "{{ .Values.image }}:{{ .Values.imageTag }}"
+        image: "{{ .Values.image }}{{- if .Values.imageTag -}}:{{- end -}}{{ .Values.imageTag }}"
```

Comment:
What if the chart does not contain an imageTag will the `{{ .Values.imageTag }}` does make an incorrect YAML file. Isn't it better to have this within the if statement?

Reply:
If the imageTag is empty the YAML will still be valid. The semicolon won't be added. Of course a user is then required to add the imageTag to the image variable. 
This gives an user the freedom to no use the imageTag variable. This is required for our SMHE pipeline as this pipeline updates all imageTag values.